src/crypto_tools/paillier/zk/mta.rs

```diff
@@ -121,7 +122,7 @@ impl ZkSetup {
         v: &BigNumber,
         w: &BigNumber,
     ) -> k256::Scalar {
-        let e = k256::Scalar::from_digest(
+        let e = <k256::Scalar as Reduce<k256::U256>>::from_be_bytes_reduced(
```

Comment:
I looked at `from_be_digest` as an alternative, but there's an extra trait dependency there that I wasn't able to figure out